### PR TITLE
updating hard decline messages

### PR DESCRIPTION
**Pull request by RafaelBruno**
Updating hard decline messages

### CODE REVIEW

File: src/ebanx.js [hunk 1]
```diff
@@ -70,9 +70,9 @@ EBANX.errors = (function () {
       'pt_BR': {
```
Comment (danielnass) [inline src/ebanx.js]: @gpressutto5 @andercampanha @RafaelBruno what you think to init separate files? This looks like a good path to start

Comment (gpressutto5) [reply src/ebanx.js]: 👍

Comment (andercampanha) [reply src/ebanx.js]: 👍

Comment (RafaelBruno) [reply src/ebanx.js]: Good Ideia, let's start the organization for here :)

**Pull request merged**